### ISSUE TITLE
Sleep latency low

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11537]
**Preferred term label:**
Sleep latency low 
https://www.ncbi.nlm.nih.gov/medgen/C2674747
https://www.ncbi.nlm.nih.gov/medgen/C3280268

**Synonyms**
Multiple sleep latency test < 8 minutes

**Definition (free text, please give PubMed ID)**
https://www.sleepfoundation.org/sleep-studies/multiple-sleep-latency-test-mslt

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Narcolepsy

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Are we talking about an abnormal multiple sleep latency test?
https://pubmed.ncbi.nlm.nih.gov/31277864/
Or about a clinical observation/report of patient?

**Comment by MickeySegal** [comment]
My intention was to include either.  It is important to have the non-lab history term to facilitate progress in the clinical setting before moving to lab tests.
